Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterInformationLegalHelper.kt

```diff
@@ -0,0 +1,34 @@
+@file:JvmName("FormatterInformationLegalHelper")
+
+package de.rki.coronawarnapp.util.formatter
+import android.view.View
+import java.util.Locale
+
+/**
+ * Language based format visibility
+ *
+ * @param defaultLanguageEnglishOrGerman
+ * @param isContactFormView
+ * @return
+ */
+fun formatVisibilityLanguageBased(defaultLanguageEnglishOrGerman: Boolean?, isContactFormView: Boolean?): Int {
+    if (defaultLanguageEnglishOrGerman!!) {
```

Comment:
if checking for true and not null you can also use `defaultLanguageEnglishOrGerman  == true` this will stop a `NullPointerException` from occuring if the value is actually null